app/src/pages/Kitchen/index.tsx

```diff
@@ -10,22 +11,39 @@ import { Badge } from '~src/components/Badge/Badge';
 import { IconButton } from '~src/components/IconButton/IconButton';
 import { findArrayIndex, replaceItemAtIndex } from '~src/utils';
 import { BadgeColor } from '~src/components/Badge/types';
+import { apiURL, serverURL } from '~src/urls';
 
 export const KitchenPage = () => {
   const [restaurantOrders, setRestaurantOrders] = useState<Order[]>([]);
   const [selectedOrder, setSelectedOrder] = useState<Order>();
 
+  const socket = io(serverURL);
+
+  useEffect(() => {
+    socket.connect();
```

Comment:
I think we don't need to connect the socket since io establishes a connection when creating the new socket.